doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -127,114 +130,122 @@ General configuration
 
 	Initialize the mesh shell. This command must be run before any other mesh command.
 
+``mesh reset-local``
+--------------------
 
-``mesh reset <addr>``
----------------------
-
-	reset the local mesh node to its initial unprovisioned state or reset a remote node and remove it from the network.
-	* ``addr``: address of the node to reset.
-
-
-``mesh lpn <value: off, on>``
------------------------------
-
-	Enable or disable Low Power operation. Once enabled, the device will turn off its radio and start polling for friend nodes. The device will not be able to receive messages from the mesh network until the friendship has been established.
-
-	* ``value``: Sets whether Low Power operation is enabled.
-
-
-``mesh poll``
--------------
-
-	Perform a poll to the friend node, to receive any pending messages. Only available when LPN is enabled.
-
-
-``mesh ident``
---------------
-
-	Enable the Proxy Node Identity beacon, allowing Proxy devices to connect explicitly to this device. The beacon will run for 60 seconds before the node returns to normal Proxy beacons.
-
+	Reset the local mesh node to its initial unprovisioned state.
 
-``mesh dst [destination address]``
-----------------------------------
+``mesh target dst [destination address]``
```

Comment:
Should commands in documentation now be grouped the same way as in code? E.g. create a separate subsection for `mesh target` commands explaining why they are needed.

Reply:
Hmm, I will take a look at it